src/VisualStudio/Core/Def/SymbolSearch/VisualStudioSymbolSearchService.cs

```diff
@@ -76,18 +76,9 @@ private void OnPackageSourcesChanged(object sender, EventArgs e)
 
         protected override void StartWorking()
         {
-            // Kick off a database update.
-            var sources = _installerService.PackageSources;
-
             // Always pull down the nuget.org index.  It contains the MS reference assembly index
             // inside of it.
-            var allSources = sources.Concat(new PackageSource(
-                SymbolSearchUpdateEngine.NugetOrgSource, source: null));
-
-            foreach (var source in allSources)
-            {
-                Task.Run(() => UpdateSourceInBackgroundAsync(source.Name));
-            }
+            Task.Run(() => UpdateSourceInBackgroundAsync(SymbolSearchUpdateEngine.NugetOrgSource));
```

Comment:
why this change?  the intent here was to allow other parties to participate in the search.  we coded our end up to make that possible, did the other interested parties retract their interest in playing in tihs space with us?

Reply:
This is covered by the commit message for 45822d18ea8d1c4a1d271a3c64144747d6bebdac. I will file a follow-up issue to implement OOP caching.

Reply:
i don't think i quite understood this bit...  specifically, i don't quite get teh justification of:

>Package information for other sources cannot be cached until the
operation runs out-of-process.

Could you just clarify that bit?

Reply:
The background caching results in a bunch of JIT operations and/or assembly loads even for cases where the user is not triggering this feature. RPS flagged the regression and this fixes it.

Reply:
> The background caching results in a bunch of JIT operations and/or assembly loads even for cases where the user is not triggering this feature. RPS flagged the regression and this fixes it.

thanks much!